src/config/dynamic-theme-fixes.config

```
@@ -12219,6 +12226,9 @@ body,
 
 nixos.org
 
+INVERT
+a::after
```

Where can I observe this change?

---

```suggestion
header a[href="/"]::after
```
Just to ensure that it won't touch any other `a` elements that use the psuedo-elements.